### PR TITLE
Fix broken link for DSL reference

### LINKED ISSUE #16785: Broken link in `Copy` error message when upgrading to Gradle 7.0
Upgrading a project to Gradle 7 yields the following error message: 

```
Entry META-INF/gradle-plugins/com.osacky.fulladle.properties is a duplicate but no duplicate handling strategy has been set. Please refer to https://docs.gradle.org/7.0/dsl/org.gradle.api.file.CopySpec.html#org.gradle.api.file.CopySpec:duplicatesStrategy for details. 
```

The error message has a broken link.

It should probably be: https://docs.gradle.org/7.0/dsl/org.gradle.api.tasks.Copy.html#org.gradle.api.tasks.Copy:duplicatesStrategy
or
https://docs.gradle.org/7.0/javadoc/org/gradle/api/file/CopySpec.html#setDuplicatesStrategy-org.gradle.api.file.DuplicatesStrategy-

Here is a build scan of the error:
https://scans.gradle.com/s/gbgserqxkezai/console-log#L9

Steps to reproduce:
Check out: https://github.com/runningcode/fladle
Run `./gradlew wrapper --gradle-version 7.0`

### PR DESCRIPTION
**Pull request by melix**
Fixes #16785

### CODE REVIEW

**Comment by melix** [comment]
@bot-gradle test this

File: subprojects/base-services/src/main/java/org/gradle/api/internal/DocumentationRegistry.java [hunk 1]
```diff
@@ -44,6 +44,11 @@ public String getDslRefForProperty(Class<?> clazz, String property) {
         return String.format("https://docs.gradle.org/%s/dsl/%s.html#%s:%s", gradleVersion.getVersion(), className, className, property);
     }
 
+    public String getDslRef(Class<?> clazz, String anchor) {
+        String className = clazz.getName();
+        return String.format("https://docs.gradle.org/%s/dsl/%s.html#%s", gradleVersion.getVersion(), className, anchor);
```
Comment (wolfs) [inline subprojects/base-services/src/main/java/org/gradle/api/internal/DocumentationRegistry.java:49]:
```suggestion
        return String.format("https://docs.gradle.org/%s/javadoc/%s.html#%s", gradleVersion.getVersion(), className, anchor);
```

I don't think there is a `CopySpec` DSL entry, only a Javadoc entry. This is what you are looking for, right: https://docs.gradle.org/current/javadoc/org/gradle/api/file/CopySpec.html#getDuplicatesStrategy-- ?

**Comment by melix** [comment]
@bot-gradle test this

File: subprojects/base-services/src/main/java/org/gradle/api/internal/DocumentationRegistry.java (hunk 1 above)
Comment (melix) [reply subprojects/base-services/src/main/java/org/gradle/api/internal/DocumentationRegistry.java]: Darn I need my glasses ... Fixed by using the DSL doc for `Copy`.

**Comment by melix** [comment]
@bot-gradle test this

**Comment by melix** [comment]
@bot-gradle test and merge

**Comment by melix** [comment]
@bot-gradle test and merge

**Pull request merged**